stablehlo/compatibility/TODO.txt

```diff
@@ -0,0 +1,9 @@
+This file is not to be submitted. Just needed to log the work still to be done somewhere central.
+
+1. Address FIXME comments
+2. Error if target version is greater than current version.
+3. Error if target version is less than minimum supported version.
+4. Validate target version string.
+5. Add versioned dialect type forks.
+6. Add bytecode for Versioned StableHLO
+7. Add versioned CHLO ops
```

Comment:
Versioned CHLO is the lowest priority of these work items - we can push it into 2023.

Reply:
Sounds good. Will note that in the GH issue.